service/src/main/kotlin/fi/espoo/evaka/reservations/ReservationControllerCitizen.kt

```diff
@@ -110,7 +111,7 @@ class ReservationControllerCitizen(
         if (body.dateRange.start.isBefore(dateNow()))
             throw BadRequest("Cannot mark absences for past days")
 
-        if (!listOf(OTHER_ABSENCE, PLANNED_ABSENCE, SICKLEAVE).contains(body.absenceType))
+        if (!listOf(OTHER_ABSENCE, PLANNED_ABSENCE, SICKLEAVE, FREE_ABSENCE).contains(body.absenceType))
             throw BadRequest("Invalid absence type")
```

Comment:
Reporting holidays (`FREE_ABSENCE`) should only be allowed from the holiday form and the user input needs to be validated so that it matches the free period options. Come to think of it, refactoring the separate requests per child to a single request should probably be done now.